site/src/components/ConfirmationAlert.tsx

```diff
@@ -0,0 +1,48 @@
+import { FunctionComponent, ReactNode } from "react";
```

Comment:
This is unused - it is to confirm deleting sub-schemas. Will either use it or remove it in a follow up.